run.py

```diff
@@ -0,0 +1,7 @@
+import os, subprocess, locale
+cwd = os.getcwd()
+def cmd(execute):
+    return subprocess.call(execute.format(cwd).split())
```

Comment:
Sure. Can refactor this to be more bullet-proof and nice with a proper library later on (that wouldn't be call[...])